modules/@apostrophecms/doc/index.js

```diff
@@ -252,6 +253,13 @@ module.exports = {
       async enableCollection() {
         self.db = await self.apos.db.collection('aposDocs');
       },
+      // Detect whether the database is brand new (zero documents).
+      // This can't be done later because after this point init()
+      // functions are permitted to insert documents
+      async detectNew() {
+        const existing = await self.db.find({}).project({ _id: 1 }).limit(1).toArray();
```

Comment:
Couldn't you `count`? Or `toCount`?

Reply:
I could have used [looks at latest recommended mongo APIs] `countDocuments`, sure. Works fine though; how about we open a ticket to refine this as it's well tested and can make it into this release?

Reply:
I don't think that ticket would ever get done. Would that be a big lift to change? It seems like you're doing extra work to get to the same place as a method that already exists to do this.